docs/proposals/ose-rebase.md

```diff
@@ -0,0 +1,39 @@
+## Rebase OSE on Origin
+
+Today, code in OSE lands by fetching and merging Origin manually. OSE-specific changes
+are scattered in the repo and it's hard to reason about any conflicts. This doc proposes
+moving all the changes in OSE that diverge from Origin into a set of commits that
+will sit on top of the Origin code[0] and a Jenkins job that will be rebasing master and
+all active release branches on a daily basis.
+
+There are two types of commits that need to be carried on top of Origin: 
+1. long-term carries related to branding and release tooling (master and release branches)
+2. CVEs (release branches only)
```

Comment:
[from @kargakis]
Not sure I undestand this. Is the separation of Origin CI from OSE CI not a maintainace cost per se? Or is it less expensive to distribute the load brought up by testing/merging PRs across the two repos? I would expect us to have a clear flow as to how commits flow from one repo to the other. The less bidirectional this thing is the better.